blog/lead-maintainer-report-march-2020.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: layout.html
+---
+
+<% set('title', 'BCHN lead maintainer report 2020-03-14') %>
+<% set('date', '14 March 2020') %>
+<% set('author', 'freetrader') %>
+
+Hello Bitcoin Cash and Bitcoin Cash Node user communities!
+
+More than 2 weeks have passed since the initial release (v0.21.0) of Bitcoin Cash Node, and today I want to take some time to report back to you on what's happened since, and the activities we've been busy on and are working/planning right now.
+
+These reports will hopefully become a much more regular publication as things settle down a bit and we get our project processes up to speed.
+
+### Project uptake
+
+Since release, at least 3 mining pools have mined blocks signaling support of Bitcoin Cash Node (and implying their disagreement with the IFP):
+
+*   pool.bitcoin.com
+
+*   ASICseer
+
+*   SBI Crypto
+
+Several sites have sprung up to monitor adoption of `/BCHN/` signaling miners / pools:
+
+*   [https://bitcoincash.network/voting/](https://bitcoincash.network/voting/) (at time of writing, 6.9% over last 2016 blocks in period 624961 - 626976)
```

Comment:
In the rendered HTML, the space before `(at the time` is dropped which looks bad

Reply:
Weird. I removed the "https" prefix from the link and it worked. Prob some issue with the `techy` dependency.

It should be fixed now.